src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsIpc/IpcCommands.cs

```diff
@@ -51,12 +51,15 @@ internal enum DumpCommandId : byte
     internal enum ProfilerCommandId : byte
     {
         AttachProfiler = 0x01,
+        StartupProfiler = 0x02,
     }
 
     internal enum ProcessCommandId : byte
     {
         GetProcessInfo = 0x00,
         ResumeRuntime  = 0x01,
         GetProcessEnvironment = 0x02,
+        GetEnvironmentVariable = 0x03,
```

Comment:
According to this [change](https://github.com/dotnet/runtime/commit/d156b9e58d7b09f742d4553bb5a94657cf7e28ae), the GetEnvironmentVariable command was ultimately not added and the SetEnvironmentVariable command has a value of 0x03

Reply:
ah, good catch!  @davmason is out on vacation, but I can update the PR for him if he doesn't see this.

Reply:
Thanks Justin, that's what I get for waiting too long between merging the runtime changes and updating this PR